packages/pill/src/component.tsx

```diff
@@ -6,7 +6,7 @@ import { pill as ccPill } from '@warp-ds/css/component-classes'
 export function Pill(props: PillProps) {
   return (
     <div
-      role='img'
+      role={props.role === '' ? undefined : props.role}
```

Comment:
Isn't it enough to have `role={props.role}` here? Same with `aria-label` in the next line. The reason why we had such check in other components that accepted aria-label and role properties was because we'd default to some value that should be possible to reset/override. Here we're not defaulting to anything, so I think we can make it simple :)

Reply:
ah ok! That makes sense! :)